src/coreclr/gc/gc.cpp

```diff
@@ -25501,6 +25502,11 @@ bool gc_heap::change_heap_count (int new_n_heaps)
                 hp->fix_allocation_contexts (TRUE);
             }
 
+            if (unified_gen0_bricks_cleared && (hp->gen0_bricks_cleared == FALSE))
```

Comment:
Do we have to check `unified_gen0_bricks_cleared` to set `FALSE`?

Reply:
we don't have to. it just saves having to check other heaps' gen0_bricks_cleared flag if it's already false.